plugins/tests/test_mapcss_parsing_evalutation.validator.mapcss

```diff
@@ -0,0 +1,162 @@
+/* Disclaimer: 
+This document only serves to contain test cases to evaluate
+the parsing and evaluation of mapcss files, and is therefore
+not supposed to be used in JOSM for validation of OSM data */
+
+
+node[x=0] {
+  throwWarning: "test #1740";
+	assertMatch: "node x=0";
```

Comment:
Can you fix the indentation of this file please.

Reply:
Oops, didn't notice it. Fixed now